kedro-datasets/RELEASE.md

```diff
@@ -1,6 +1,8 @@
 # Upcoming Release
```

Comment:
let's fix it in separate PR

Reply:
I wasn't able to replicate the issue with pre-commit. Could you let me know how you're running pre-commit, because for me it wasn't generate so many changes in different files when trying to commit this dataset? You can also remove that commit and continue without it: https://github.com/kedro-org/kedro-plugins/pull/810/commits/06a543912dff052d74380fcb5e2045515d807f15